packages/core/next.config.js

```diff
@@ -17,8 +20,9 @@ const nextConfig = {
     loaderFile: './src/components/ui/Image/loader.ts',
   },
   i18n: {
-    locales: [storeConfig.session.locale],
-    defaultLocale: storeConfig.session.locale,
+    locales: Object.keys(localesSettings.locales),
+    defaultLocale: localesSettings.defaultLocale,
+    localeDetection: false,
```

Comment:
[Disabling Automatic Locale Detection](https://nextjs.org/docs/pages/guides/internationalization#disabling-automatic-locale-detection) because we aren't going to deal with it now.